manifests/amandahosts.pp

```diff
@@ -9,7 +9,6 @@
   realize(Concat["${amanda::params::homedir}/.amandahosts"])
 
   concat::fragment { "amanda::amandahosts/${title}":
-    ensure  => $ensure,
```

Comment:
does this now still work with older versions of concat or do we need to bump to lower limit?

Reply:
For what's worth, I've tested my Puppet repository with puppetlabs/concat 1.2.5 and puppetlabs/xinetd 2.0.0, and it works just fine.

These are the [xinetd changes](https://forge.puppet.com/puppetlabs/xinetd/changelog) since 3.0.0:
* Puppet version requirement lower bound to 4.7.1
* $xinetd::service::server now defaults to undef

The Amanda module always defines a $xinetd::service::server, so we're good.

These are the [xinetd changes](https://forge.puppet.com/puppetlabs/concat/changelog) since 3.0.0:
* All changes in this release are for enabling rubocop, alongside the module then being converted over to the PDK.
* Rubocop has been implemented.
* SLES 10, Windows 2003 R2 and OSX 10.9 removed as supported.
* OSX 10.10 and 10.11 added as supported.
* stdlib dependency changed to 4.13.1
* supported Puppet version changed to 4.7.

Seems to be okay for the Amanda module.